lib/exprengine.cpp

```diff
@@ -971,6 +971,43 @@ static bool isNonOverlapping(ExprEngine::ValuePtr v1, ExprEngine::ValuePtr v2)
     return false;
 }
 
+static bool conditionAlwaysFalse(ExprEngine::ValuePtr condValue, ExprEngine::DataBase *dataBase)
+{
+    // TODO: This is not finished
+    if (auto v = std::dynamic_pointer_cast<ExprEngine::BinOpResult>(condValue))
+        return v->isAlwaysFalse(dataBase);
+    if (auto v = std::dynamic_pointer_cast<ExprEngine::IntRange>(condValue))
+        return v->isEqual(dataBase, 0);
```

Comment:
this only determines that the value might be false, `isEqual` is a misleading method name.

Reply:
Ah, I see. Thanks, that was unexpected.